pkgs/development/ocaml-modules/minttea/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildDunePackage
+, fetchurl
+, mdx
+, riot
+, tty
+}:
+
+buildDunePackage rec {
+  pname = "minttea";
+  version = "0.0.1";
+
+  minimalOCamlVersion = "5.1";
+
+  src = fetchurl {
+    url = "https://github.com/leostera/minttea/releases/download/${version}/minttea-${version}.tbz";
+    hash = "sha256-Ca0wa6cYs+cuQr2igB7ZhtQ0Dtk=";
+  };
+
+  propagatedBuildInputs = [
+    mdx
+    riot
+    tty
```

```suggestion
    tty
```
there is no tty package, did you mean to add coreutils?

---

Perhaps -- let me figure out how to build locally, that will help me figure out what's going on here.

---

Looks like `tty` isn't published in `nixpkgs` yet either -- I can open up another PR to do that, as well!

---

> Looks like tty isn't published in nixpkgs yet either -- I can open up another PR to do that, as well!

It is it's in the gnucoreutils package.

> Perhaps -- let me figure out how to build locally, that will help me figure out what's going on here.

You can use `nix-build -A ocamlPackages.minttea` it will give you a `./result/` directory if the run is successful

---

Got it -- thanks! Yes, looks like this builds successfully now (I think!)